src/App.js

```diff
@@ -28,27 +90,36 @@ function App() {
         </button>
         <form className="flex flex-col w-9/12 max-w-screen-sm">
           <input
```

Comment:
It's good practice to add a `name` attribute to form fields, [check here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#name). Also, it's required if you want to use FormData